code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -1,5 +1,21 @@
 #define DEFAULT_DOOMSDAY_TIMER 4500
 
+GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
+		/obj/machinery/field/containment,
+		/obj/machinery/power/supermatter_shard,
+		/obj/machinery/dominator,
+		/obj/machinery/doomsday_device,
+		/obj/machinery/nuclearbomb,
+		/obj/machinery/nuclearbomb/selfdestruct,
+		/obj/machinery/nuclearbomb/syndicate,
+		/obj/machinery/syndicatebomb,
```

Comment:
Actually my apologies, I had assumed that they could (for some reason I distinctly remember them being able to.)

AI units are not able to interface with the timers on bombs.